tesla_ble_mqtt/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## 0.0.11
+
+### Changed
+
+- No need to provide MAC addresses anymore for presence detection
```

Comment:
This is true with the Expect ble-presence but not with the current sh script. At least not as of now...

There are a few approaches that we can use to do it. The simplest one would be to send to the addon's log the bluetoothctl output until we matches all VIN (BLE Local Name) part of the list. The end user would copy / paste the MAC address. Doesn't require the end user to download an app on his smartphone and figure out where to look.

But let me think about this ... I'll get back to you.